app/models/card.rb

```diff
@@ -1,7 +1,8 @@
 class Card < ActiveRecord::Base
+  belongs_to :user
   scope :pending, -> { where("review_date <= ?", Time.now).order("RANDOM()") } 
 
-  validates :original_text, :translated_text, :review_date,   presence: true
+  validates :original_text, :translated_text, :review_date, :user ,  presence: true
```

Comment:
Line is too long. [83/80]<br>Unnecessary spacing detected.<br>Space found before comma.